<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoAppOnQtClass</class>
 <widget class="QMainWindow" name="TodoAppOnQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TodoAppOnQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>4</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>4</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taskLineEdit">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>34</y>
      <width>581</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Input a new task</string>
    </property>
   </widget>
   <widget class="QListWidget" name="taskListWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>64</y>
      <width>581</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TodoAppOnQt.qrc"/>
 </resources>
 <connections/>
</ui>
